### PR TITLE
Feature/display top ranked books

### LINKED ISSUE #3: Display Top Books on Main Page
- Create a TopBookContainer component
- Create a BookCard component
- Display top book content on page

### PR DESCRIPTION
**Pull request by megschuetz**
**[Project Ticket](https://github.com/users/mikeharrison57/projects/5#card-84506363)**

**Describe the changes made to the codebase:**

- Create the book cards container and book cards components 
- Passed/manipulated data in new components
- Add simple styling to main page components

**How was it tested?**

- Npm start and looking at the application in the browser. 

**Type of Change**

- [ ] Bug Fix
- [X] New Feature
- [ ] Refactor

**Issues**

- closes #3 

**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new bugs

**Emoji of how you feel submitting this PR (optional):**

- Emoji: 😎

### CODE REVIEW

File: src/TopBooks/TopBooks.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+const TopBooks = ({ genres }: Props) => {
+  console.log(genres)
+
+
+
+  const numberOneBooks = genres.bookLists.map((list) => {
+   return (
+ 
+    <BookCard book={list.books[0]}
```
Comment (catlynbowles) [inline src/TopBooks/TopBooks.tsx:18]: I love the portrayal of the top books ! 🐩

File: src/TopBooks/TopBooks.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.books-container {
+  display: grid;
+  grid-gap: 9rem;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
```
Comment (catlynbowles) [inline src/TopBooks/TopBooks.css:4]: Dynamic grid for the winnnn 🐍

**Review by catlynbowles — APPROVED** [review]

**Review by orlov-n — APPROVED** [review]
This code is just mind blowing!

**Review by mikeharrison57 — APPROVED** [review]
Wonderful job setting up the main page book display Megan. This looks amazing! I'm excited to dive into some routing tomorrow for our genre pages.

**Pull request merged**